### DISCUSSION TITLE
How to get markdown output?

### DISCUSSION THREAD

**Discussion by kasem-sm** [Milkdown/milkdown#790] [Q&A]
This is dumb question but I didn’t find any code sample. I’m new to TypeScript and React.

**Comment by ahamelers** [comment]
You can get it using a util command in editor.action(): https://milkdown.dev/docs/api/utils#get-markdown-fn-ctx-ctx-string

Or on change using the listener plugin: https://milkdown.dev/docs/api/plugin-listener
